### ISSUE TITLE
partial view

### ISSUE THREAD

**Issue by MoonshineSG** [jacobwgillespie/plex-sync#35]
is there a way to sync the partial view status (`viewOffset`) ?

**Comment by jacobwgillespie** [comment]
Hey!  Sorry, I'm not entirely familiar with `viewOffset` - do you have any more information about what you'd like it to do?

**Comment by MoonshineSG** [comment]
If I'm not wrong, `viewOffset` indicates the current progress of the partially watched episodes or movies...

**Comment by jacobwgillespie** [comment]
Oh, interesting - yeah, that would be useful.  I'll take a look next time I have some availability.  👍

**Comment by MoonshineSG** [comment]
and since you're there... when you have time... an interesting (for me) feature would be "mine", "hers" "ours" sync accounts.... the dynamics of it might be quite complex, but maybe worth exploring...

**Comment by jacobwgillespie** [comment]
> "mine", "hers" "ours" sync accounts

How would that work?  As it stands right now you can read and write (or just read or just write) from any source to any other source, so you could, for instance, sync the "ours" user to each of your personal users:

```bash
Server IP: 10.0.0.1
Section ID: 5
Your Plex Token: XXXXXXXX
Her Plex Token: YYYYYYYY
Shared Account Plex Token: ZZZZZZZZ

# Sync anything from shared to personal accounts by reading from the shared account
# and writing to the two personal accounts:
$ plex-sync ZZZZZZZZ@10.0.0.1/5,r XXXXXXXX@10.0.0.1/5,w YYYYYYYY@10.0.0.1/5,w

# For multiple sections, just run multiple commands:
$ plex-sync ZZZZZZZZ@10.0.0.1/8,r XXXXXXXX@10.0.0.1/8,w YYYYYYYY@10.0.0.1/8,w
$ plex-sync ZZZZZZZZ@10.0.0.1/16,r XXXXXXXX@10.0.0.1/16,w YYYYYYYY@10.0.0.1/16,w
```

**Comment by MoonshineSG** [comment]
how about reverse... if both are watched, "ours" is watched ?

**Comment by jacobwgillespie** [comment]
Ah, yes, so that's where it would have to be smart beyond what it can do now, but could definitely be implemented as an option...

I'm starting to push the limits of this command-line syntax of specifying servers, I think, and will eventually probably have to switch to JSON or YAML for configuring the sync settings.

**Comment by MoonshineSG** [comment]
yes, if you want "smart" sync, it needs a bit more than command line params... priority levels, conflict resolution rules... 
not an easy task...
